### PR TITLE
VSTS branding

### PR DESCRIPTION
**Pull request by wpschaub**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Yeoman-extension-code-generator/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Change team services to Visual Studio Team Services in descriptions
- Change repo name to generator-vsts-extension

@ALM-Rangers/Yeoman-extension-code-generator

### CODE REVIEW

**Review by willsmythe — APPROVED** [review]

**Review by mikaelkrief — APPROVED** [review]

File: generators/hub/templates/readme.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This has been updated to be inline with the [M109](https://www.visualstudio.com/
 ###For further information, please consult the following websites:
 - The VSTS Extensions page on [MSDN](https://www.visualstudio.com/en-us/integrate/extensions/overview)
 - The VSTS Extensions sample repo on [GitHub](https://github.com/Microsoft/vso-extension-samples)
```
Comment (nschonni) [inline generators/hub/templates/readme.md]: https://github.com/Microsoft/vsts-extension-samples

File: generators/widget/templates/readme.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This has been updated to be inline with the [M109](https://www.visualstudio.com/
 ###For further information, please consult the following websites:
 - The VSTS Extensions page on [MSDN](https://www.visualstudio.com/en-us/integrate/extensions/overview)
 - The VSTS Extensions sample repo on [GitHub](https://github.com/Microsoft/vso-extension-samples)
```
Comment (nschonni) [inline generators/widget/templates/readme.md]: https://github.com/Microsoft/vsts-extension-samples

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "generator-team-services-extension",
```
Comment (nschonni) [inline package.json:2]: Probably need to go through https://docs.npmjs.com/cli/deprecate to change the name again :wink:

File: generators/hub/templates/readme.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#Welcome to Visual Studio Team Services Extensions!
+﻿#Welcome to Visual Studio Team Services Extensions!
```
Comment (nschonni) [inline generators/hub/templates/readme.md:1]: Aside: These headings are broken on GH. The need the space after the `#`

Comment (wpschaub) [reply generators/hub/templates/readme.md:1]: Fixed, thank you.

File: package.json (hunk 3 above)
Comment (wpschaub) [reply package.json:2]: We'll defer this breaking change.

**Pull request merged**